src/main.py

```diff
@@ -109,7 +109,11 @@ def get_dataset_by_id(
     responses={
         status.HTTP_200_OK: {
             "description": "Successful response. Returns all the editions for the dataset.",
+<<<<<<< HEAD
```

Comment:
some left over merge kurft here you'll need to resolve.

I won't tag them all but I think this file (and test_version) slipped in.